### PR TITLE
Update docker.md

### PR DESCRIPTION
**Pull request by teofanis**
Update outdated docker-examples link

### CODE REVIEW

File: website/docs/docker.md [hunk 1]
```diff
@@ -224,7 +224,7 @@ Please note that for any of the above docker commands you need to have docker in
 
 There is a separate repository that hosts multiple configurations to compose Docker images with `verdaccio`, for instance, as reverse proxy:
 
-[https://github.com/verdaccio/docker-examples](https://github.com/verdaccio/docker-examples)
+[https://github.com/verdaccio/docker-examples](https://[github.com/verdaccio/docker-examples](https://github.com/verdaccio/verdaccio/tree/master/docker-examples))
```
Comment (juanpicado) [inline website/docs/docker.md:227]: I think this `[` shouldn't be there, right?

**Review by juanpicado — COMMENTED** [review]
Thanks for taking care :) just small markdown issue there and we can go.

File: website/docs/docker.md (hunk 1 above)
Comment (teofanis) [reply website/docs/docker.md:227]: Yes ! Nice catch, fixing it now :)

**Review by juanpicado — APPROVED** [review]
Thanks !

**Pull request merged**